platforms/r3-corda-ent/images/pki.dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM corda/enterprise-pkitool:1.2-zulu-openjdk8u242
```

Comment:
Is this changing of working dir needed?

Reply:
No, this is probably not necessary.
I will re-create the image on our registry and test this out.

Reply:
Fixed

Reply:
The 'outdated' tag is not showing here because the review comment was on one line above the actual fix.
But in the changed files, the new pki dockerfile is present.